apps/admin/package.json

```diff
@@ -28,7 +28,8 @@
     "react": "16.14.0",
     "react-dom": "16.14.0",
     "regenerator-runtime": "^0.13.5",
-    "theme": "^0.1.0"
+    "theme": "^0.1.0",
+    "tslib": "^2.4.0"
```

Comment:
@endymion1818 just to confirm: is `tslib` dependency still necessary? I know you mentioned that it's our rmwc forks that were causing issues, is that related?

Reply:
@Pavel910 unfortunately yes, Im afraid we still get that issue with the `spreadArray` function we identified before without explicitly adding this line.

Reply:
In that case, I assume we'll need to add this dependency as part of the project upgrade we're preparing for 5.30. Hopefully it's the only thing that needs changing in existing user projects. Have you tried upgrading from 5.29 ?

Reply:
Ah, no haven't tested that. I'll run through it in the morning.